make container fails to build images

<!-- PLEASE FILL THESE FIELDS, IT REALLY HELPS THE MAINTAINERS OF SearXNG -->

**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**

0ce0d957b11e70298ef4c1a0aeefe0d9f72f77ec

**How did you install SearXNG?**

git clone

**What happened?**

```
ERROR: failed to solve: failed to parse stage name "localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder": invalid reference format: repository name ($CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME) must be lowercase

```

**How To Reproduce**

git clone, make container
**Expected behavior**

images to be built

**Screenshots & Logs**

```
> [+] Building 18.0s (10/10) FINISHED                                                                             docker:default
>  => [internal] load build definition from builder.dockerfile                                                              0.0s
>  => => transferring dockerfile: 941B                                                                                      0.0s
>  => [internal] load metadata for ghcr.io/searxng/base:searxng-builder                                                     1.0s
>  => [internal] load .dockerignore                                                                                         0.0s
>  => => transferring context: 97B                                                                                          0.0s
>  => [builder 1/5] FROM ghcr.io/searxng/base:searxng-builder@sha256:e3e8b93d16bf8096056512432065be677674274bcad0b57ba6c1a  8.2s
>  => => resolve ghcr.io/searxng/base:searxng-builder@sha256:e3e8b93d16bf8096056512432065be677674274bcad0b57ba6c1acfb5e95f  0.0s
>  => => sha256:e3e8b93d16bf8096056512432065be677674274bcad0b57ba6c1acfb5e95fd55 780B / 780B                                0.0s
>  => => sha256:a89cf01138a2989e183cb11c017948a7a2db2b93a809d438de40a89e15bf304e 481B / 481B                                0.0s
>  => => sha256:11eafb4d3a1018d8d4ab8b04708f5b96df5d68df0be8f0f9c7427b3b5143e25e 649B / 649B                                0.0s
>  => => sha256:c5fc9178e8df14eb5ae97b2715fef348a94371fb266b6bda2048f4771b3d34f4 144.10MB / 144.10MB                        2.4s
>  => => extracting sha256:c5fc9178e8df14eb5ae97b2715fef348a94371fb266b6bda2048f4771b3d34f4                                 5.7s
>  => [internal] load build context                                                                                         0.4s
>  => => transferring context: 18.25MB                                                                                      0.4s
>  => [builder 2/5] COPY ./requirements*.txt ./                                                                             1.9s
>  => [builder 3/5] RUN --mount=type=cache,id=uv,target=/root/.cache/uv set -eux;     uv venv;     uv pip install --no-man  4.2s
>  => [builder 4/5] COPY ./searx/ ./searx/                                                                                  0.3s
>  => [builder 5/5] RUN set -eux;     python -m compileall -q ./searx/;     touch -c -t 202509111910.27 ./searx/settings.y  1.2s
>  => exporting to image                                                                                                    1.1s
>  => => exporting layers                                                                                                   1.1s
>  => => writing image sha256:1b2c5790d50b418dfa998c59b7e8c9b6988a082277f64a8546c85d173fe39b25                              0.0s
>  => => naming to localhost/searxng/searxng:builder                                                                        0.0s
> CONTAINER Image "builder" built
> [+] Building 0.0s (2/2) FINISHED                                                                                docker:default
>  => [internal] load build definition from dist.dockerfile                                                                 0.0s
>  => => transferring dockerfile: 1.89kB                                                                                    0.0s
>  => CANCELED [internal] load metadata for ghcr.io/searxng/base:searxng                                                    0.0s
> dist.dockerfile:9
> --------------------
>    7 |     COPY --chown=searxng:searxng --from=localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder /usr/local/searxng/searx/ ./searx/
>    8 |     COPY --chown=searxng:searxng ./container/ ./
>    9 | >>> COPY --chown=searxng:searxng --from=localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder /usr/local/searxng/version_frozen.py ./searx/
>   10 |
>   11 |     ARG CREATED="0001-01-01T00:00:00Z"
> --------------------
> ERROR: failed to solve: failed to parse stage name "localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder": invalid reference format: repository name ($CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME) must be lowercase
> ERROR: container.build exit with error (1)
> make: *** [Makefile:92: container.build] Error 1

**Additional context**
<!-- Add any other context about the problem here. -->
$ docker version
Client:
 Version:           28.0.4
 API version:       1.48
 Go version:        go1.25.1 X:nodwarf5
 Git commit:        b8034c0ed70494a90c133461d145cd072d920d7c
 Built:             Sat Sep 13 03:13:31 2025
 OS/Arch:           linux/amd64
 Context:           default

Server:
 Engine:
  Version:          28.0.4
  API version:      1.48 (minimum version 1.24)
  Go version:       go1.24.5
  Git commit:       6430e49a55babd9b8f4d08e70ecb2b68900770fe
  Built:            Sun Aug  3 17:42:42 2025
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v2.1.1
  GitCommit:        cb1076646aa3740577fafbf3d914198b7fe8e3f7
 runc:
  Version:          1.3.0
  GitCommit:        4ca628d1d4c974f92d24daccb901aa078aad748e
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad007797e0dcd8b7126f27bb87401d224240
```

--- a/container/dist.dockerfile
+++ b/container/dist.dockerfile
@@ -1,10 +1,11 @@
-FROM ghcr.io/searxng/base:searxng AS dist
-
 ARG CONTAINER_IMAGE_ORGANIZATION="searxng"
 ARG CONTAINER_IMAGE_NAME="searxng"
 
-COPY --chown=searxng:searxng --from=localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder /usr/local/searxng/.venv/ ./.venv/
-COPY --chown=searxng:searxng --from=localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder /usr/local/searxng/searx/ ./searx/
+FROM localhost/$CONTAINER_IMAGE_ORGANIZATION/$CONTAINER_IMAGE_NAME:builder AS builder
+FROM ghcr.io/searxng/base:searxng AS dist
+
+COPY --chown=searxng:searxng --from=builder /usr/local/searxng/.venv/ ./.venv/
+COPY --chown=searxng:searxng --from=builder /usr/local/searxng/searx/ ./searx/
 COPY --chown=searxng:searxng ./container/ ./
 #COPY --chown=searxng:searxng ./searx/version_frozen.py ./searx/
 

--- a/utils/lib_sxng_container.sh
+++ b/utils/lib_sxng_container.sh
@@ -62,6 +62,10 @@ container.build() {
     esac
     info_msg "Selected platform: $platform"
 
+    if [ "$container_engine" = "docker" ] && ! docker buildx version &>/dev/null; then
+        die 42 "docker buildx is not installed: https://docs.docker.com/go/buildx/"
+    fi
+
     pyenv.install
 
     (

